### ISSUE TITLE
Add support for NULL values

### ISSUE THREAD

**Issue by YPermitin** [lukasz-pyrzyk/BinaryFormatter#19]
This test doesn't pass, because NULL values aren't serializing.

```
List<string> value = new List<string>();
value.Add("lorem ipsum");
value.Add(null);
value.Add("Кто не ходит, тот и не падает.");

BinaryConverter converter = new BinaryConverter();
byte[] bytes = converter.Serialize(value);
List<string> deserializedValue = converter.Deserialize<List<string>>(bytes);

Assert.Equal(value, deserializedValue);
```

**Comment by lukasz-pyrzyk** [comment]
Added do the milestone 1.1.1

**Comment by YPermitin** [comment]
Fixed in the PR #49.

**Comment by YPermitin** [comment]
In the Develop branch.
